pages/index.js

```diff
@@ -1,8 +1,24 @@
 import Head from "next/head";
 import Banner from "../components/atoms/Banner.js";
 import { ActionButton } from "../components/atoms/ActionButton";
+import { SearchBar } from "../components/atoms/SearchBar";
+import { useState } from "react";
 
 export default function Home() {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+
+  //Handler for input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
+
+  //Handler for submit
+  function onSearchSubmit(e) {
+    e.preventDefault();
+    alert("Search value: " + searchText);
```

Comment:
We can remove this for the time being, but it looks like it works 👍

Reply:
Since this will be a site-wide search, we will want this somewhere in something like a `Layout` component instead of per-page, but for now it's all good (we only have 1 page)